src/routes/Upgrade/UpgradeViewContainer.js

```diff
@@ -75,7 +76,7 @@ function validateMnemonic(bip32XPublicKey: ?string) {
     derivationPath,
   }: UpgradeMnemonicFormFieldValues): UpgradeMnemonicFormFieldErrors => {
     if (!bip32XPublicKey) {
-      throw new Error(t`WalletDataError`)
+      throw new WalletInvalidDataError('', 'bip32XPublicKey is empty')
```

Comment:
Why is it not `WalletEmptyAddressError` and `WalletEmptyBIP32XPubError`? How do we decide when we use new error class and when we use string description? How do we check errors for specific descriptions?

Reply:
this is called "inconsistency" when we expect to get specific valid data for specific type of wallet, but this data doesn't exist